package.json

```diff
@@ -9,21 +9,25 @@
     "lint": "next lint"
   },
   "dependencies": {
+    "bootstrap": "^5.3.3",
```

Comment:
won't need bootstrap in the meantime so we can remove these packages (react-router-hash-link, react-bootstrap)